src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2909,6 +2909,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return Me.AnonymousTypeManager.ConstructAnonymousTypeSymbol(descriptor)
         End Function
 
+        Protected Overrides Function CommonCreateBuiltinOperator(
+                name As String,
+                returnType As ITypeSymbol,
+                leftType As ITypeSymbol,
+                rightType As ITypeSymbol,
+                isChecked As Boolean) As IMethodSymbol
```

Comment:
Should we throw if this is true? there are no checked operators in VB, I think?

Reply:
VB def has checked operators.  And it passes true here in several cases.  for example:

```vb
' for unary
Checked AndAlso operandType.IsIntegralType() AndAlso
op = UnaryOperatorKind.Minus
```

```vb
' for binary
Checked AndAlso leftType.IsIntegralType() AndAlso
(op = BinaryOperatorKind.Multiply OrElse
op = BinaryOperatorKind.Add OrElse
op = BinaryOperatorKind.Subtract OrElse
op = BinaryOperatorKind.IntegerDivide)
```

Note: it does not used the 'checked' name in this case (as that's about calling these new operators).